ciao_contrib/runtool.py

```diff
@@ -2782,7 +2782,7 @@ def make_tool(toolname):
 parinfo['destreak'] = {
     'istool': True,
     'req': [ParValue("infile","f","Input dataset/block specification",None),ParValue("outfile","f","Output dataset/block specification",None)],
-    'opt': [ParValue("max","s","streak threshold  syntax:  m  OR  m:m:m:m",None),ParRange("max_rowloss_fraction","r","Maximum fraction of avg streaks/node/frame",5.0e-5,0.3e-4,1),ParValue("num_sigma","r","Sigma value for determining streak threshold",1.0),ParValue("filter","b","Discard tagged events",True),ParValue("mask","s","Filter to select candidate streak events",'[status=0,grade=0,2:4,6]'),ParValue("ccd_id","s","CCD ID to filter",'8'),ParValue("ccd_col","s","CCD ID column name",'ccd_id'),ParValue("node_col","s","Node ID column name ('none' for single node)",'node_id'),ParValue("exptime","r","frame time (s) (reads EXPTIME if no pos. value given)",-1),ParValue("countfile","f","filename for event row-count distribution",None),ParValue("fracfile","f","filename for cumulative streak contam function",None),ParValue("timefile","f","filename for exposure time lost per row",None),ParRange("verbose","i","Debug Level(0-5)",0,0,5),ParValue("clobber","b","Clobber existing file",False)],
+    'opt': [ParValue("max","s","streak threshold  syntax:  m  OR  m:m:m:m",None),ParRange("max_rowloss_fraction","r","Maximum fraction of avg streaks/node/frame",5.0e-5,3e-05,1),ParValue("num_sigma","r","Sigma value for determining streak threshold",1.0),ParValue("filter","b","Discard tagged events",True),ParValue("mask","s","Filter to select candidate streak events",'[status=0,grade=0,2:4,6]'),ParValue("ccd_id","s","CCD ID to filter",'8'),ParValue("ccd_col","s","CCD ID column name",'ccd_id'),ParValue("node_col","s","Node ID column name ('none' for single node)",'node_id'),ParValue("exptime","r","frame time (s) (reads EXPTIME if no pos. value given)",-1),ParValue("countfile","f","filename for event row-count distribution",None),ParValue("fracfile","f","filename for cumulative streak contam function",None),ParValue("timefile","f","filename for exposure time lost per row",None),ParRange("verbose","i","Debug Level(0-5)",0,0,5),ParValue("clobber","b","Clobber existing file",False)],
```

Comment:
Humm .... this looks liks a bug in `destreak.par` where the `min` is greater than the `max`